src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarBidAction.test.tsx

```diff
@@ -31,155 +36,578 @@ describe("ArtworkSidebarBidAction", () => {
           artwork(id: "auction_artwork") {
             ...ArtworkSidebarBidAction_artwork
           }
+          me {
+            ...ArtworkSidebarBidAction_me
+          }
         }
       `,
       mockData: {
-        artwork: response,
+        artwork: response.artwork,
+        me: response.me,
       } as ArtworkSidebarBidAction_Test_QueryRawResponse,
     })
   }
 
   describe("for closed auction", () => {
     it("does not display anything", async () => {
-      const wrapper = await getWrapper(ArtworkFromClosedAuction)
+      const wrapper = await getWrapper({
+        artwork: ArtworkFromClosedAuction,
+        me: NotIDVedUser,
+      })
 
       expect(wrapper.html()).toBe(null)
     })
   })
 
   describe("for auction preview", () => {
     it("and not registered bidder", async () => {
-      const wrapper = await getWrapper(ArtworkFromAuctionPreview)
+      const wrapper = await getWrapper({
+        artwork: ArtworkFromAuctionPreview,
+        me: NotIDVedUser,
+      })
 
       expect(wrapper.text()).toContain("Register to bid")
     })
 
     it("with bidder registration pending approval", async () => {
-      const artwork = merge(
-        {},
-        ArtworkFromAuctionPreview,
-        BidderPendingApproval
-      )
-      const wrapper = await getWrapper(artwork)
+      const artwork = merge(ArtworkFromAuctionPreview, BidderPendingApproval)
```

Comment:
@dleve123 - we really should be using `cloneDeep` here since a merge doesn't return a new object which can lead to mutation problems.

Reply:
We discussed this and determined that because we're not mutating the fixture that this isn't needed.